test/utils/emojisCache.spec.js

```diff
@@ -18,7 +18,11 @@ describe('emojisCache', () => {
   })
 
   it('should match CACHE_PATH', () => {
-    expect(CACHE_PATH).toMatchSnapshot()
+    if (process.platform !== 'win32') {
+      expect(CACHE_PATH).toMatchSnapshot()
+    } else {
+      expect('/Users/Test/.gitmoji/gitmojis.json').toMatchSnapshot()
```

Comment:
From a test point of view, this doesn't makes sense at all

It's like doing `expect(1).toMatchSnapshot()` or `expect(1).toEqual(1)`